File: src/main/java/br/com/g1bet/repository/TimeRepository.java
package br.com.g1bet.repository;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import br.com.g1bet.model.Time;

@Repository
public interface TimeRepository extends JpaRepository<Time, Long> {
    List<Time> findAllByNomeContainingIgnoreCase(String nome);

    Boolean existsByNome(String nome);

}

File: src/main/java/br/com/g1bet/repository/TimeRepository.kt
package br.com.g1bet.repository

import br.com.g1bet.model.Time
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface TimeRepository : JpaRepository<Time?, Long?> {
    fun findAllByNomeContainingIgnoreCase(nome: String?): List<Time?>?
    fun existsByNome(nome: String?): Boolean?
}